packages/opc-input-chip/src/opc-input-chip.scss

```diff
@@ -0,0 +1,50 @@
+@import '@one-platform/opc-styles';
+
+$LOCAL: inputChip;
+
+:host {
+  display: block;
+}
+
+.opc-input-chip {
+  border: 1px solid;
+  border-color: opc-get-var(borderColor, $opc-color--black-300);
+  padding: 4px 1rem;
+  min-height: 50px;
+  border-radius: 5px;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+
+  .opc-input {
+    flex: 1;
+    margin: 4px;
```

Comment:
can u use global css variable here?

Reply:
u can use calc function to make it 4px

Reply:
yeah created a map to hold reusable units. I tried using the following interpretation but it wasn't working

```scss
$input-chip-units: (
  margin: 4,
  border-width: 1,
);

.opc-input {
  // to string interpolation with unit as px/rem etc
 // this doesnt work 
  margin: #{map-get($input-chip-units, margin)px}
}

```
So Created a map with units

```scss
$input-chip-units: (
  margin: 4px,
  border-width: 1px,
);

```